modules/check_in/spec/services/v2/chip/service_spec.rb

```diff
@@ -260,4 +260,84 @@
       end
     end
   end
+
+  describe '#confirm_demographics' do
```

Comment:
I feel these kind of exceptions thrown from downstream service calls can be handled under a spec for `connection` method defined [here](https://github.com/department-of-veterans-affairs/vets-api/blob/a172f902481c1a18e8734b09293663ac67a68a21/modules/check_in/app/services/v2/chip/client.rb#L134-L134).  

It can be made generic so we don't need to hard code downstream service messages in rspecs.

Reply:
Synced offline with @randomsync, instead of including downstream failure tests in service_spec, we can have a test covered in request_spec.  Will address it in separate PR for updating request_spec to unblock integration testing.